src/handler/Command.js

```
@@ -9,38 +9,42 @@ class Command extends Toggleable {
    * @param {string} [options.info] - Information about this command
    * @param {string} [options.usage] - Usage of this command
    * @param {boolean} [options.guildOnly] - Whether the command can only be used inside a guild
+   * @param {Array<string>} [options.contributors] - List of github users who contributed to the command
    */
   constructor(name, options) {
     super();
 
     this.name = name;
 
-    if (!Array.isArray(options.aliases)) {
+    this.aliases = options.aliases || [];
```

This should be reverted as we want to throw an error when a falsy value is passed.

---

I think that change is fine because not every command might have an alias, and we shouldn't require it to have an alias.

---

Passing an empty array would've be valid with the check we had before.

---

Passing an empty array yes, but not passing one at all wouldn't be valid.

---

This applies to what I said above.

> I think the command options should always be explicitly provided. It makes it easy to see everything about the command, without relying on default values that are set somewhere else. This is why the command options don't have default values, and always throw an error when they're not provided.